src/plot/axis/axis-title.js

```diff
@@ -23,6 +23,32 @@ import React from 'react';
 import {ORIENTATION} from 'utils/axis-utils';
 
 const {LEFT, RIGHT, TOP, BOTTOM} = ORIENTATION;
+const transformation = (width, height) => ({
+  [LEFT]: {
+    x: 16,
```

Comment:
Ok, so I'm not entirely clear on your suggested fix, should I pull out the assumed base `em`  as a constant and multiply that?